backend/benchmark.py

```
@@ -0,0 +1,431 @@
+
+        for tool_call in message.tool_calls or []:
+            if tool_call.get("name") != "vectordb_search":
+                continue
+
+            call_id = tool_call.get("id")
+
+            for followup in messages[i + 1:]:
+                if isinstance(followup, ToolMessage) and followup.tool_call_id == call_id:
+                    try:
+                        return json.loads(followup.content)
+                    except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)